src/v2/ion/responseTransformer.js

```diff
@@ -35,8 +35,8 @@ const isObject = x => _.isObject(x);
  */
 const getFirstLevelObjects = (resp) => {
   const result = {};
-  _.each(resp, (val, key) => {
-    // if key is remediation we dont do any transformation
+  _.each(resp, (val = {}, key) => {
```

Comment:
In [some cases](https://github.com/okta/okta-idx-js/blob/master/src/v1/makeIdxState.js#L41), this can be `undefined`. Adding a default assignment here for safety.